nemo/collections/asr/models/hybrid_rnnt_ctc_bpe_models.py

```diff
@@ -134,6 +134,13 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
 
         # setting the RNNT decoder as the default one
         self.cur_decoder = "rnnt"
+        self.epoch_count = 0
+
+    def on_train_epoch_end(self):
```

Comment:
this may be simplified by setting `cfg.num_epochs=1` instead (and possibly incrementing by 1 every time you run another training loop)